IBX-8493: Ibexa Experience Product guide (#2448)
ibexa/documentation-developer: docs/ibexa_products/experience.md

---
title: Ibexa Experience

> description: Discover Ibexa Experience and its essential features, capabilities and benefits.

Author: In my opinion, it won't look good if all three editions will have exactly the same description. 
Yes, it's the same message in another words. But unification won't look good in this situation.
I rewrote it in another way.

month_change: true
---

# Ibexa DXP Experience edition product guide

## What is [[= product_name_exp =]]

[[= product_name_exp =]] is an [[= product_name =]] edition that focuses on the customer.

> It offers smooth consumer journey and great online experience. In everything you do, it places your clients first.

Author: It's mentioned in 'Technical backstage' section.
Core features are described, no need to list the differences.

## Availability

To start using [[= product_name_exp =]] you need to purchase a product license.
For more information, see [[[= product_name_exp =]] license pricing](https://www.ibexa.co/products/pricing?tab=2).
You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).

## How it works

### Technical backstage

With an active lincese, you can start [installation process](install_ibexa_dxp.md) through the Composer.

[[= product_name_exp =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.

Version control systems and environment variables allow you to deploy your extensions and settings on several environments.

> [[= product_name_exp =]] is built on [[[= product_name_headless =]]](headless.md). It means that Experience edition includes all bundles, APIs, and [features that comes with Headless](headless.md#core-features).

Author: I'd put it higher, but after Symfony part.

## Capabilities and benefits

With [[= product_name_exp =]] you can focus on the customers and treat each of them as a VIP.
It has everything you need to offer a transformative digital experience, from developing new websites, portals, eye-catching landing pages, and personalized product suggestions, to managing SEO strategies across several locations.

### Core features

[[= product_name_exp =]] comes with a variety of new features to help you create great customer experience.

> #### Page Builder

Author: I've changed the order, I'd keep them in separate sections.

> #### Page Builder

Reviewer: Yes, I didn't mean to create a single paragraph for them - thanks!

[[= product_name_exp =]] brings new visual tool - [Page Builder](page_builder_guide.md). With Page Builder you can design and modify pages. It involves more than just maintaining, it also includes developing pages, tailored content, and landing pages that are perfectly targeted. Content is composed using ready-to-use elements - blocks, properly configured and customized.

> You also need to select a layout. The layout that is selected has a significant impact on how the content pieces in the drop zones are arranged.

Author: "Need" is not as _strong_ as "have to". Anyway, it's true that you need to do it - yes, it gives a lot of flexibility, but even if it's one-zone page, you still need to choose it.

[Form Builder](form_builder_guide.md) is an intuitive tool that allows you to transform user engagement on your website.
With this tool, you can design, deploy, and manage online forms quickly.
You can create a variety of forms consisting of different fields, including sign-up forms, survey, or questionnaire.
Additionaly, you can monitor and manage the information obtained from website visitors and adjust your forms if needed.

![Form Builder](form_builder.png)

#### Publish Later

You can take complete control of where and when your content blocks are visible to your predefined audiences and [schedule content publication]([[= user_doc =]]/content_management/schedule_publishing/).
[[= product_name_exp =]] comes with a Publish Later feature that allows you to schedule personalized content and reach different user groups at optimal dates and times to boost performance.
What's more, you can turn specific content pages and blocks on and off to meet the needs of your marketing campaigns and promotions.
Publish Later feature combined with Page Builder allows you to show all changes that you plan to the page for the future - just use the slider to see all the upcoming changes.

> #### Customer Data Platform (CDP)

Author: I've added a sentence saying that it's an add-on.
Regarding segmentation - "CDP helps you to improve engagement, (...) by **_segmenting_** your audience (...)" - I guess it's enough here, there is a link to CDP guide.

> #### Customer Data Platform (CDP)

Reviewer: I would still expand this into two sections: Segmentation and CDP.

- Segmentation allows you to segment the userbase, which you can later use directly in Page Builder (with Targeting block) and with Personalization (https://doc.ibexa.co/projects/userguide/en/latest/personalization/segment_management/) to tailor the content seen by visitors.

This is available ouf of the box, for everyone who uses the Experience edition.

Then, if someone decides to invest deeper into this area, CDP awaits with even more possibilities (but requires an additional deal)

---
title: Ibexa Experience
description: Discover all the main features, functions, and advantages of the Ibexa Experience.
month_change: true
---

# Ibexa DXP Experience edition product guide

## What is [[= product_name_exp =]]

[[= product_name_exp =]] is an [[= product_name =]] edition that focuses on the customer.
It offers smooth consumer journey and great online experience. In everything you do, it places your clients first.
With Experience edition you can empower Editors to quickly create new pages or personalized content, and improve their daily work.
It also provides tools for using segmentation and targeting, and thanks to the features and intergrations can be widely used in B2B.

## Availability

To start using [[= product_name_exp =]] you need to purchase a product license.
For more information, see [[[= product_name_exp =]] license pricing](https://www.ibexa.co/products/pricing?tab=2).
You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).

## How it works

### Technical backstage

> With an active license, you can start [installation process](install_ibexa_dxp.md) through Composer.

Author: I'd keep previous version.

> With an active license, you can start [installation process](install_ibexa_dxp.md) through Composer.

Reviewer 2: I specifically dislike "through Composer". Composer is a tool, so... "with"

Version control systems and environment variables allow you to deploy your projects and settings on several environments, such as Ibexa Cloud.

## Capabilities and benefits

With [[= product_name_exp =]] you can focus on the customers and treat each of them as a VIP.
It has everything you need to offer a transformative digital experience, from developing new websites, portals, eye-catching landing pages, and personalized product suggestions, to managing SEO strategies across several locations.

### Core features

[[= product_name_exp =]] comes with a variety of new features to help you create great customer experience.

#### Page Builder

[[= product_name_exp =]] brings new visual tool - [Page Builder](page_builder_guide.md). With its intuitive and plain interface you can design and modify pages, even without having advanced technical skills. It also includes developing pages, tailored content, and landing pages that are perfectly targeted. Content is composed using ready-to-use elements - blocks, properly configured and customized.

> When creating a page you also need to select a layout. It has a significant impact on how the content pieces in the drop zones are arranged.

Author: I don't like using 'Tip' in Product Guides. I'd keep previous version.

With this tool, you can design, deploy, and manage online forms quickly.
You can create a variety of forms consisting of different fields, including sign-up forms, surveys, or questionnaires.
Additionaly, you can monitor and manage the information obtained from website visitors and adjust your forms if needed.

![Form Builder](form_builder.png)

#### Site Factory

[Site Factory ](site_factory.md) is a site management interface, integrated with the Back Office. It enables you to configure new sites without leaving the administration interface and editing SiteAccess configuration.
With this feature you can create and deploy multiple websites at lightning speed and at scale.
It allows you to manage expenses and resources while industrializing your web presence. Additionally, together with localized information and tailored product catalogs and prices, it helps you to quickly enter new markets.

#### Customizable Dashboard

Starting from Experience edition of [[= product_name =]] you can [customize the Dashboard]([[= user_doc =]]/getting_started/dashboard/work_with_dashboard/#customize-dashboard).

> You can create customized Dashboard using Dashboard Builder. Dashboard then can be tailored to your specific needs - it contains a set of widgets selected by the user. You can easily preview the sections that you use more often and omit the less significant ones.

Author: I'd keep previous version of the last sentence, "promote" doesn't fit here (in my opinion).

What's more, you can turn specific content pages and blocks on and off to meet the needs of your marketing campaigns and promotions.
Publish Later feature combined with Page Builder allows you to show all changes that you plan to the page for the future - just use the slider to see all the upcoming changes.

#### Customer Portal

Through the [[= product_name =]] Customer Portal and customer management that comes with it, you can manage company accounts, establish new ones, and communicate with your partners in a personalized space.
With the help of this feature, you can create customized areas that give users a smooth, integrated experience and provide them access to a variety of resources, apps, and services from a single point of entry.
Using this tool, your customers can change their organization details, invite and see members, self-register, and more.

#### Segments

[Segmentation](https://doc.ibexa.co/en/latest/administration/admin_panel/segments_admin_panel/) allows you to split up the user base.
Using segments you can display specific content to specific Users and tailor the content that the visitors can see.

> The Targeting and Dynamic targeting blocks available in the Page Builder use them right out of the box.
> Segments work also with the [Personalization](personalization_brochure.md), where you can assign users to different recommendation groups and create advanced logic with operators to get best segmentation for your audience.

Reviewer: ```suggestion
The Targeting block is available in the Page Builder and you can use it right out of the box.
Segments work also with the [Personalization](personalization_brochure.md), where you can assign users to different recommendation groups and create advanced logic with operators to devlier the best recommendations for your audience.
```
1) There are some words missing in the first sentence
1) The Dynamic targeting block requires the Personalization integration (which is mentioned a sentence later)
2) Personalization delivers recommendations and I'd focus on that- the segmentation has to be done before that (manually or with CDP)

Additionally, you can manage and analyze campaigns, evaluate customer data, and identify the best ways to improve performance.
With [[= product_name_cdp =]] you can store and manage large volumes of customer data in a structured manner. This central data storage supports business growth with a scalable infrastructure, helping to futureproof your business.

![CDP](cdp.png)

#### Ibexa Engage

Another add-on available for Experience and Commerce edition, is [Ibexa Engage](ibexa_engage.md).
To use it, you must make arrangements with [[= product_name_base =]] to define the initial configuration, and then get and set up a user account.
Ibexa Engage is a data collection tool. It gives you the ability to use the [Qualifio](https://qualifio.com/) tools to engage your audiences. You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns and collect important data.
To promote your campaign, you can add a Campaign block to a page in Page Builder or embed a campaign within the Rich Text field by using a Campaign custom tag.

![Ibexa Engage](ibexa_engage.png)

### Use cases

> With [[= product_name_exp =]] your customers are the main focus of all that you do.
> It makes it simpler than ever to create the different touchpoints that your customers have with your brand, giving you the ability to guide them through your significant business procedures. 

Reviewer 2: ```suggestion
With [[= product_name_exp =]], your customers are the main focus of all that you do.
It makes it simpler than ever to create the different touchpoints that your customers have with your brand, giving you the ability to guide them through your significant business procedures. 
```

#### Build new pages and integrated forms

> User interface of [[= product_name_exp =]] is intuitive and plain. With its new features - Page and Form Builder - you can build new pages or forms quickly. Page Builder comes with predefined layouts, blocks, and templates. Thanks to the Site Factory, you can publish as many sites as you want - there's no limit. Form Builder also comes with ready-to-use elements. You can integrate your custom forms and surveys into the website. What's more, you can reuse content from existing sites to new pages.

Reviewer 2: ```suggestion
[[= product_name_exp =]]'s user interface is intuitive and user-friendly.
With its new features, Page Builder and Form Builder, you can design pages and forms quickly and efficiently.
Page Builder comes with predefined layouts, blocks, and templates to streamline your design process, while Form Builder provides ready-to-use elements for easy form creation.
Once you've built your custom forms and surveys, you can integrate them into your website and reuse content from existing sites on new ones.
Plus, with Site Factory, you can publish as many sites as you like, there are no limits.
```

#### Target customers in their preferred channels

> If you want to make your product attractive to the customers, you need to know, that each of them is unique and special. You should then remember to tailor your marketing strategy to their needs and preferences. 
> [[= product_name_exp =]] allows you to deliver personalized content and recommendations through different channels.
> With segmentation you can create audiences to distribute specific content in the right channels, at the right time.
> Available add-ons give you even more possibilities - you can analyze customer behaviours, use interactive content, and collect important data.

Reviewer 2: ```suggestion
To make your products attractive, you must remember each of your customers is unique and special, and tailor your marketing strategy to their needs and preferences. 
[[= product_name_exp =]] allows you to deliver personalized content and recommendations through different channels.
With segmentation, you can define audiences that you will reach with specific content through the right channels, at the right time.
And add-ons give you even more possibilities: you can analyze customer behaviours, use interactive content, and collect important data.
```

#### Build your future

> Planning and scheduling is a part of management. With scheduling tools available in Experience edition, you can prepare content publishing timetables, schedule how your website can evolve, and test or preview it before publication. You can use editorial calendars for an easier collaboration.

Reviewer 2: ```suggestion
Planning and scheduling is a part of management.
With scheduling tools available in Experience edition, you can prepare content publishing timetables, plan the evolution of your website, and test or preview it before publication.
Additionally, you can use editorial calendars for an easier collaboration.
```
